nucliadb/nucliadb/ingest/orm/abc.py

```diff
@@ -50,23 +50,25 @@
 
 class ShardCounter(BaseModel):
     shard: str
-    fields: int
+    resources: int
```

Comment:
mh I am not sure about this one.
Resources can have many fields mapped to it, so I think it makes sense to shard based on the number of fields